Dockerfile

```diff
@@ -6,6 +6,7 @@ ENV VIRAL_CLASSIFY_PATH=$INSTALL_PATH/viral-classify \
 	PATH="$PATH:$MINICONDA_PATH/envs/env2/bin"
 
 COPY requirements-conda.txt requirements-conda-env2.txt $VIRAL_CLASSIFY_PATH/
+COPY classify/megablast_nt_trimmed_down.sh classify/retrieve_top_blast_hits_LCA_for_each_sequence.pl $VIRAL_CLASSIFY_PATH/
```

Comment:
```suggestion
```

Don't need this line... and in fact, probably undesirable to have it here so early because any edits to these perl scripts will cause the conda layers to rebuild. The contents of `classify/` should get copied by line 15 below.